SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -640,6 +657,12 @@ void USpatialReceiver::CleanupDeletedEntity(Worker_EntityId EntityId)
 	NetDriver->RemoveActorChannel(EntityId);
 }
 
+void USpatialReceiver::ClearRPCsOnEntityCreation(Worker_EntityId EntityId)
```

Comment:
I still see this function in the Receiver. Was it moved to the Sender locally?

Reply:
Moved to Sender